evergreen/config.yml

```diff
@@ -218,19 +232,25 @@ functions:
               [[ -d coverage_data ]] || mkdir coverage_data
               [[ -d coverage_binaries ]] || mkdir coverage_binaries
               export LLVM_PROFILE_FILE="$(pwd)/coverage_data/${task_name}-%p.profraw"
-              EXECUTABLE_PATH=$(find build -name ${test_executable_name} -type f)
-              EXECUTABLE_PATH=$(./evergreen/abspath.sh $EXECUTABLE_PATH)
+              EXECUTABLE_FILE="$(find build -name "${test_executable_name}" -type f)"
+              EXECUTABLE_PATH="$(./evergreen/abspath.sh "$EXECUTABLE_FILE")"
```

Comment:
invalid nested quotes here

Reply:
This is allowed for `$()` operations in bash shell scripts - the inner quotation marks prevent [globbing](https://www.shellcheck.net/wiki/SC2086) of the variable inside the command and the same with the outer quotation marks.

Reply:
ah cool! TIL